src/app/helpers/fake-server/fake-server.ts

```diff
@@ -0,0 +1,18 @@
+import { User } from 'src/app/models/user';
+import { AuthUser } from 'src/app/models/auth-user';
+
+export class FakeServer {
+    private static fakeUserStorage:Array<AuthUser> = [
+        {login:'admin',password:'admin',token:'adminfaketoken228'},
```

Comment:
Use the same formatting everywhere: choose for yourself - 2 spaces or 4. I would go with two - it's easier to look through the code with nesting